modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -487,4 +496,43 @@ public static void closeAll(Collection<? extends AutoCloseable> closeables) thro
     public static void closeAll(AutoCloseable... closeables) throws Exception {
         closeAll(Arrays.asList(closeables));
     }
+
+    /**
+     * Short date format pattern for log messages in "quiet" mode.
+     * Only time is included since we don't expect "quiet" mode to be used
+     * for longer runs.
+     */
+    private static final SimpleDateFormat SHORT_DATE_FMT = new SimpleDateFormat("HH:mm:ss");
+
+    /**
+     * @param log Logger.
+     * @param msg Message.
+     */
+    public static void dumpStack(IgniteLogger log, String msg) {
+        String reason = "Dumping stack.";
+
+        var err = new Exception(msg);
+
+        if (log != null)
+            log.error(compact(reason), err);
```

Comment:
Why do you need to compact the reason if it's a constant string? Something's wrong with this method.

Reply:
It is a copy-paste from Ignite 2 method.

Reply:
It doesn't explain why you replace substrings that don't exist, can you please fix this code?